packages/schemas/src/schemas.ts

```diff
@@ -177,4 +177,5 @@ export const FileUpload = Type.Array(
 
 export const Null = Type.Null({ description: "An empty value" });
 
-export const RichText = Type.String({ description: "A string of rich text" });
+// Rich text is validated via prosemirror, so we allow Any here
```

Comment:
Could you update this comment to mention that we validate the structure of this field when it changes?

Reply:
updated! though it only validates on submit, not on every change